discovery-handlers/onvif/src/discovery_impl.rs

```diff
@@ -310,7 +314,7 @@ pub mod util {
         scopes_filters: Option<&FilterList>,
         timeout: Duration,
     ) -> Result<Vec<String>, anyhow::Error> {
-        let mut broadcast_responses = std::collections::HashSet::new();
+        let mut broadcast_responses = Vec::new();
```

Comment:
That can be a HashSet too, but agree it doesn't provide much value at this scope.